### PR TITLE
i18n some static options from the OSD

### PR DESCRIPTION
**Pull request by McGiverGim**
This add translation to all the "static" options of the OSD and other missing strings.

I need to translate too the warnings and statistics, but this will be left for other PR.

### CODE REVIEW

**Comment by Docteh** [comment]
I don't know that translating PAL and NTSC label is a good idea.

I'd set the description a bit more clear: Option to force use of PAL on the OSD

Option for chip to automatically determine video standard.
That might need more work.

File: locales/en/messages.json [hunk 1]
```diff
@@ -4310,18 +4333,58 @@
...
+      "message": "Total armed time",
+      "description": "One of the options for the source timer"
+    },
+    "osdTimerSourceOptionLastArmedTime": {
+      "message": "Last armed time",
+      "description": "One of the options for the source timer"
+    },
+    "osdTimerSourceOptionOnArmTime": {
+      "message": "On/Armed time",
+      "description": "One of the options for the source timer"
```
Comment (Docteh) [inline locales/en/messages.json]: I think this description should describe how the option works. For context.

This option shows On Time when craft is disarmed, and Armed Time when armed.

**Comment by McGiverGim** [comment]
Yes, the descriptions are for the translators only and can be more descriptive. I translate things like PAL and NTSC because we have languages with differents alphabets (Chinese, Japanese, Russian...) In this way they can use it's own symbols.

**Comment by Docteh** [comment]
Like "ＰＡＬ" and " ＮＴＳＣ"?

**Comment by mikeller** [comment]
I guess if it turns out that PAL is PAL in all languages, and NTSC is NTSC then we have done unneeded work. But I think we have to try to see if this is the case.

**Comment by McGiverGim** [comment]
Added more detailed descriptions for the timer options as requested in the review.

I don't think we need it, but for example "NTSC" in Zulu language is "I-NTSC" according to Google :P

**Pull request merged**